_inc/client/plans/style.scss

```
@@ -207,7 +207,7 @@ $plan-features-sidebar-width: 272px;
 	}
 }
 
-.plans-mobile-notice {
+.plans-mobile-notice.dops-card {
```

Since I'm not sure if `.plans-mobile-notice` element isn't used in another context where it doesn't have to be also `.dops-card`, I'd suggest adding a separate selector instead of increasing specificity of this one, i.e.:

```suggestion
.plans-mobile-notice,
.plans-mobile-notice.dops-card {
```